uipa_org/templates/header.html

```diff
@@ -30,4 +31,50 @@
         </a>
     </li>
 </ul>
-{% endblock %}
\ No newline at end of file
+{% endblock %}
+
+{% block nav_account_login %}
+    {% if request.user.is_authenticated %}
+        <li class="nav-item dropdown account-dropdown {% block nav_account %}{% endblock %}">
+            <a class="nav-link dropdown-toggle account-dropdown-link"
+            data-bs-toggle="dropdown"
+            href="#"
+            aria-haspopup="true"
+            aria-expanded="false"
+            id="navbaraccount-link">
+                <span class="d-none d-md-none d-lg-inline">
+    Profile
```

Comment:
I think that the logged in user's first name should be kept to provide an indication as to who is logged in. This is accomplished with:

```
{% blocktrans with name=request.user.first_name|truncatechars:12 %}                               
    Profile ({{ name }})                                                                            
{% endblocktrans %} 
```

However, if using the logged in user doesn't have the First name and the Last name fields populated, the first name won't be shown.